pycontrails/physics/units.py

```diff
@@ -56,17 +57,17 @@ def ft_to_pl(h: ArrayScalarLike) -> ArrayScalarLike:
     return m_to_pl(ft_to_m(h))
 
 
-def kelvin_to_celcius(kelvin: np.ndarray) -> np.ndarray:
+def kelvin_to_celcius(kelvin: ArrayScalarLike) -> ArrayScalarLike:
```

Comment:
*Celcius is spelled wrong here --> `celsius`

Reply:
🤦 ac34d70